src/edu/csus/ecs/pc2/imports/ccs/ContestSnakeYAMLLoader.java

```diff
@@ -258,6 +258,20 @@ private void setTitle(IInternalContest contest, String title) {
 
     }
 
+
+    private void setSandboxCommand(IInternalContest contest, String sandboxCommandLine) {
+        ContestInformation contestInformation = contest.getContestInformation();
+        contestInformation.setSandboxCommandLine(sandboxCommandLine);
+        contest.updateContestInformation(contestInformation);
+    }
+    
+    private void setMemoryLimit(IInternalContest contest, Integer memoryLimit) {
```

Comment:
The declaration in ContestInformation defines memoryLimitInMeg as an int; why is the parameter here an Integer? Suggest changing it to an int.

Reply:
done. pushed